genie-client/src/integTest/java/com/netflix/genie/client/JobClientIntegrationTest.java

```diff
@@ -218,6 +218,8 @@ void canSubmitJob() throws Exception {
                     null,
                     null,
                     null,
+                    null,
+                    null,
```

Comment:
A test that uses the newly added page size would be nice.
i.e. iterate over 3 jobs using page size 1